modules/EvManager/manifest.yaml

```diff
@@ -0,0 +1,104 @@
+description: >-
+  This module implements a Car simulator that can execute charging sessions using the yeti-simulation-control interface
```

Comment:
The yeti-simulation-control interface doesn't exist anymore

Reply:
That's right, it should actually say `using the car_simulator interface`. Probably copied over from the JsEvManager module

Reply:
Yes, it was copied from the `JsEvManager`. I will quickly change that

Reply:
Fixed